packages/flutter/lib/src/cupertino/action_sheet.dart

```diff
@@ -745,6 +745,11 @@ class _RenderCupertinoAlert extends RenderBox {
              },
            );
   }
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(ColorProperty('dividerColor', dividerColor));
```

Comment:
To be clear, you mean:

```dart
properties.add(ColorProperty('dividerColor', dividerColor, defaultValue: null));
```

and that's because `ColorProperty` does not insist that it's value be non-null?

```dart
  /// Create a diagnostics property for [Color].
  ///
  /// The [showName], [style], and [level] arguments must not be null.
  ColorProperty(
```